manager/integration/framework/longhorn/__init__.py

```diff
@@ -0,0 +1 @@
+from .longhorn import from_env
```

Comment:
The same as strategy. I feel you can just have longhorn.py under the root package, or make it specific like longhorn_api.py.